examples/CustomCollectionView-Swift/Sample/MosaicCollectionViewLayout.swift

```diff
@@ -229,17 +229,6 @@ class MosaicCollectionViewLayoutInspector: NSObject, ASCollectionViewLayoutInspe
   }
   
   /**
-   * Asks the inspector for the number of supplementary sections in the collection view for the given kind.
-   */
-  func collectionView(_ collectionView: ASCollectionView, numberOfSectionsForSupplementaryNodeOfKind kind: String) -> UInt {
```

Comment:
This protocol method is deprecated and no longer called, and it calls `ASCollectionView.dataSource` which is now deprecated.